packages/arb-avm-cpp/data_storage/src/arbcore.cpp

```diff
@@ -1735,22 +1743,34 @@ bool ArbCore::handleLogsCursorRequested(ReadTransaction& tx,
     if (!log_inserted_count.status.ok()) {
         logs_cursors[cursor_index].error_string =
             log_inserted_count.status.ToString();
+        std::cerr << "logscursor index " << cursor_index
+                  << " error getting inserted count: "
+                  << log_inserted_count.status.ToString() << std::endl;
         logs_cursors[cursor_index].status = DataCursor::ERROR;
-        return true;
+        return;
     }
 
     auto current_count_result =
         logsCursorGetCurrentTotalCount(tx, cursor_index);
     if (!current_count_result.status.ok()) {
         std::cerr << "Unable to get logs cursor current total count: "
-                  << cursor_index << "\n";
-        return false;
+                  << cursor_index << std::endl;
```

Comment:
Should set `logs_cursors[cursor_index].status = DataCursor::ERROR;` in this case I think

Reply:
sure